### ISSUE TITLE
Cannot create Artifact with large key/token

### ISSUE THREAD

**Issue by thebe14** [FraunhoferISST/DataspaceConnector#672]
**Describe the bug**
I am trying to create a new Artifact entity by making a request to `POST /api/artifacts`. The `accessUrl` needs an access token, and I was told in an IDSA dev meeting that I can use the `apiKey` member of this request to specify a HTTP header to be sent along with the `accessUrl`. I am passing in this JSON in the body:

```json
{
  "title": "My first artifact",
  "accessUrl": "https://oneprovider-covid19.datahub.egi.eu/api/v3/oneprovider/data/000000000052075E67756964233235643737653830313038643239306135633735633233346666353164373137636839356261233438633936373036376233376334323839663732316135653965663831663962636839356261/content",  
  "apiKey": {
    "key": "X-Auth-Token",
    "value": "TURBeFkyeHZZMkYwMGFXOXVJR1JoZEdGb2RXSXVaV2RwTG1WMUNqQXdObVJwWkdWdWRHbG1hV1Z5SURJdmRHMXdMVEV2ZFhOeUxUTXpZbU5qWkdWaE9UaGxNekE0WXpSaU1qUmxPV1E1Tm1VMU5XVTVNamMyWTJnMk5UWXdMMkZqZEM4NU1UWmlZemN6WXpBek1EbGlPRGRrWlRObFl6a3daV0UzWlRGaE16Rm1ZMk5vTlRjd1pBb3dNREZoWTJsa0lIUnBiV1VnUENBeE5qTXlOVGcwME5qQTFDakF3TVRaamFXUWdaR0YwMFlTNXlaV0ZrYjI1c2VRb3dNR05rWTJsa0lHUmhkR0V1YjJKcVpXTjAwYVdRZ1BTQXdNREF3TURBd01EQXdOVEl3TnpWRk5qYzNOVFk1TmpReU16TXlNelUyTkRNM016YzJOVE00TXpBek1UTXdNemcyTkRNeU16a3pNRFl4TXpVMk16TTNNelUyTXpNeU16TXpORFkyTmpZek5UTXhOalF6TnpNeE16YzJNelk0TXprek5UWXlOakV5TXpNMDBNemcyTXpNNU16WXpOek13TXpZek56WXlNek16TnpZek16UXpNak00TXprMk5qTTNNekl6TVRZeE16VTJOVE01TmpVMk5qTTRNekUyTmpNNU5qSTJNelk0TXprek5UWXlOakVLTURBeVpuTnBaMjVoZEhWeVpTQjAwOUwwMFdvck9Ka2xCYlVoWGJpWHJOU0lqNWZjcjk2TmlrTHRZdkxmQXRDQW8="
  },
  "filename": "test.csv",
  "mediaType": "text/csv",
  "automatedDownload": true
}
```

This fails with an 500 error. If I use a shorter text for the token, it works. Please make room for our access tokens in the DB schema you use for artifacts. 

**To Reproduce**
1. Go to the connector's Swagger-UI, navigate to `POST /api/artifacts`
2. Click on 'Try it out' and enter the JSON from above
3. Will get error 500

**Expected behavior**
This should create the artifact and return 201.

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise
 - Browser: Opera (latest build)
 - Version (DSC): Latest from the `main` branch, runs in a container, with the in-memory database

**Comment by brianjahnke** [comment]
Hi,

key and value are currently restricted to a length of 255. This probably needs to be bumped up.

**Comment by thebe14** [comment]
It is very urgent for us, I need to use a connector with real data from our backend in a presentation in a week from now. Do you think this can be fixed easily/fast?

**Comment by brianjahnke** [comment]
Hi,

could you please try if the changes in the linked PR are enough for your requirements?

**Comment by thebe14** [comment]
Yes, this works (I can create artifacts) with tokens from my backend (the size of those is what you see in the example above). Please also check again he size of the `accessUrl` you can store, as that is also pretty long in our case (it includes an object ID from our backend). Thank you very much for the fast reaction.

**Comment by thebe14** [comment]
Please note I did not yet check if this artifact can be fetched through a consumer connector, I do not yet have code to do that. Will update this issue next week when I will get to that.

**Comment by thebe14** [comment]
This was closed, but the fix for this bug is not yet merged in the main branch, right?

**Comment by brianjahnke** [comment]
Hi,
it is already merged in the main branch and part of release v6.4.0.
